Index: bluechi-0.8.0+git20240704/config/meson.build
===================================================================
--- bluechi-0.8.0+git20240704.orig/config/meson.build
+++ bluechi-0.8.0+git20240704/config/meson.build
@@ -4,25 +4,29 @@
 # SPDX-License-Identifier: LGPL-2.1-or-later
 
 install_data(
-  install_dir : get_option('sysconfdir') / 'bluechi'
+  install_dir : get_option('sysconfdir') / 'bluechi',
 )
 
 install_data(
   'agent/agent.conf',
-  install_dir : join_paths(get_option('datadir'), 'bluechi-agent', 'config')
+  install_dir : join_paths(get_option('datadir'), 'bluechi-agent', 'config'),
+  install_tag : 'bluechi-agent',
 )
 
 install_data(
   'controller/controller.conf',
-  install_dir : join_paths(get_option('datadir'), 'bluechi', 'config')
+  install_dir : join_paths(get_option('datadir'), 'bluechi', 'config'),
+  install_tag : 'bluechi-controller',
 )
 
 install_data(
   'agent.conf.d/README.md',
-  install_dir :  join_paths(get_option('sysconfdir') / 'bluechi' / 'agent.conf.d')
+  install_dir :  join_paths(get_option('sysconfdir') / 'bluechi' / 'agent.conf.d'),
+  install_tag : 'bluechi-agent',
 )
 
 install_data(
   'controller.conf.d/README.md',
-  install_dir :  join_paths(get_option('sysconfdir') / 'bluechi' / 'controller.conf.d')
+  install_dir :  join_paths(get_option('sysconfdir') / 'bluechi' / 'controller.conf.d'),
+  install_tag : 'bluechi-controller',
 )
